openpype/lib/local_settings.py

```diff
@@ -522,6 +522,11 @@ def get_local_site_id():
 
     Identifier is created if does not exists yet.
     """
+    # override local id from environment
+    # used for background syncing
+    if os.environ.get("SITE_SYNC_LOCAL_ID"):
```

Comment:
I would change env variable name to `OPENPYPE_LOCAL_ID` as it changes local id no matter if it's site sync related.